tests/tck-build-logic/src/main/groovy/org/graalvm/internal/tck/harness/TckExtension.java

```diff
@@ -351,4 +353,62 @@ List<String> getMetadataFileList(Path directory) throws IOException {
             return foundFiles;
         }
     }
+
+    String getLatestLibraryVersion(String libraryModule) {
+        try {
+            List<String> coordinates = List.of(libraryModule.split(":"));
+            String group = coordinates.get(0);
+            String artifact = coordinates.get(1);
```

Comment:
Does this really need to be a List?
```suggestion
            String[] coordinates = libraryModule.split(":");
            String group = coordinates[0];
            String artifact = coordinates[1];
```